### PR TITLE
chore: move perfomance scale tests to other perf tests

### PR DESCRIPTION
**Pull request by janisz**
## Description

Move scale tests together with other performance tests.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### Here I tell how I validated my change

N/A

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: tests/performance/scale/tests/kube-burner/README.md
Comment (rhybrillou) [inline tests/performance/scale/tests/kube-burner/README.md:1]: Nit: this README file still references the old repository. It would be good to update the git reference.

**Review by rhybrillou — APPROVED** [review]

**Comment by mtodor** [comment]
@janisz I would suggest separating the `kube-burner` and `k6` tests. Maybe we can have a structure like this:
```
performance
|- load
|- scale
```

**Comment by janisz** [comment]
@mtodor Is it better now?

File: tests/performance/load/package.json [hunk 1]
```diff
@@ -6,11 +6,11 @@
     "private": true,
     "scripts": {
         "test": "k6 run tests/testSacScopes.js",
-        "har-to-k6": "har-to-k6",
-        "format:check": "prettier --check .",
-        "format:fix": "prettier --write .",
-        "lint:check": "eslint .",
-        "lint:fix": "eslint --fix ."
+        "har-to-k6": "../node_modules/.bin/har-to-k6",
```
Comment (mtodor) [inline tests/performance/load/package.json:9]: Why is this change required? We have moved the whole project to another directory. This should stay the same.

Comment (janisz) [reply tests/performance/load/package.json]: Good catch, fixed!

**Review by mtodor — APPROVED** [review]
